driver-mct/cime_config/config_component.xml

```diff
@@ -1085,7 +1085,7 @@
 
   <entry id="GLC_GRID">
     <type>char</type>
-    <valid_values>gland20,gland10,gland5,gland5UM,gland4,mpas.aisgis20km,mpas.gis20km,mpas.ais20km,mpas.gis1to10km,null</valid_values>
+    <valid_values>gland20,gland10,gland5,gland5UM,gland4,mpas.aisgis20km,mpas.gis20km,mpas.ais20km,mpas.gis1to10km,mpas.gis1to10kmR2,null</valid_values>
```

Comment:
flagging one more place we might want to delete old gis1to10km grid reference.  

We might also want to remove the CISM meshes

Reply:
Good catch. I removed the CISM meshes. Leaving the 1to10 variants for now until we decide what to do there.

Reply:
Ok, sounds good